### PR TITLE
Title roads

### PR DESCRIPTION
**Pull request by zixuan75**

### CODE REVIEW

**Comment by zixuan75** [comment]
What did you mean?

**Comment by alax99** [comment]
my title road

File: javascript/node-master.js [hunk 1]
```diff
@@ -106,50 +137,308 @@ class user{
...
+      setSettingLogs(settingLogs){
+        this.settingLogs=settingLogs
+      }
+      getUserLogs(){
+        if (this.userLogs){
+          return this.userLogs
+        }
+      }
+      setUserLogs(userLogs){
+        this.userLogs=userLogs
```
Comment (zixuan75) [inline javascript/node-master.js]: Is this rendered by default?

File: javascript/node-master.js [hunk 2]
```diff
@@ -106,50 +137,308 @@ class user{
...
+            this.userLogs=user.logs
+            this.userBriefLogs=user.briefLogs
+            setSettingLogs(this.contents, this.logs)
+          }else{
+            this.briefLogs=user.briefLogs
+            return this.settingLogs
+          }
+        }
+        setSettingLogs(log, define){
+          this.settingLogs=log+define
```
Comment (zixuan75) [inline javascript/node-master.js]: I think this is only for one variable.

File: javascript/node-master.js [hunk 3]
```diff
@@ -106,50 +137,308 @@ class user{
...
+      event.preventDefault({
+        mixins: getMaster,
+        getData(){
+          return this.mixins
+        },
+        addMixin(){
+          newLogs.string.getMixins([1, 2, 3, 4]).addMixins(this.mixins)
+        },
+        getMaster(){
+          return newLogs.string.getMixins([1, 2, 3, 4]).getMaster()
```
Comment (zixuan75) [inline javascript/node-master.js]: Get master just return: `return getMaster`.

File: javascript/node-master.js [hunk 4]
```diff
@@ -106,50 +137,308 @@ class user{
...
+  if (!this.props){
+    this.props={
+      configure: newLogs.string.getMixins([1, 2, 3, 4])
+    }
+  }else{
+    var config=this.props.configure
+    if (this.props.configure){
+      newLogs.string.getMixins([1, 2, 3, 4]).addMixins(this.props.configure)
+    }else{
+      newLogs.string.setMixins(mixin.mixins)
```
Comment (zixuan75) [inline javascript/node-master.js]: Why you want to set a mixin?

Comment (zixuan75) [reply javascript/node-master.js]: Yes! The config is using the 'getMaster' attribute.

**Comment by zixuan75** [comment]
Let's go! @alax99

File: javascript/node-master.js (hunk 1 above)
Comment (zixuan75) [reply javascript/node-master.js]: what did you mean

File: javascript/node-master.js [hunk 5]
```diff
@@ -106,50 +137,332 @@ class user{
...
+        newLogs.string.emptyConfigure=undefined
+      }
+    }
+  }
+  if (mixin.mixins){
+    newLogs.string.setMixins(mixin.mixins)
+  }else{
+    newLogs.string.setMixins(getMaster)
+  }
+}
```
Comment (zixuan75) [inline javascript/node-master.js]: Mixin to station your title road.

File: javascript/node-plus.js [hunk 6]
```diff
@@ -6,16 +6,41 @@ var plus=class{
...
   }
   waring(){
-    var masterWaring=this.setPlus(this.plus)
+    this.setPlus(this.model)
     var waring=this.getPlus()
     this.garble=masterWaring
     this.waring=waring
     this.toPlusSet=this.waring+this.garble
   }
+  getPlus(){
```
Comment (zixuan75) [inline javascript/node-plus.js]: What did you mean?

File: javascript/node-master.js (hunk 5 above)
Comment (zixuan75) [reply javascript/node-master.js]: What did you mean?

**Comment by zixuan75** [comment]
Hello ---

**Comment by zixuan75** [comment]
Preview **of the reply**. @alax99

**Comment by zixuan75** [comment]
@alax99 Please full the pull requests opening.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
This is the vent comparison for #41.

**Comment by zixuan75** [comment]
@alax99 You should run this command to fix the problem:
`$ python -m SimpleHTTPServer`

**Comment by zixuan75** [comment]
@alax99 What did you mean?

**Comment by zixuan75** [comment]
Everything must go for :1st_place_medal:!

**Comment by zixuan75** [comment]
@alax9:
`You are not a member of any organizations.`

**Comment by zixuan75** [comment]
:1234: .....

**Comment by zixuan75** [comment]
:calendar: :atom: :atom_symbol:

**Comment by zixuan75** [comment]
:e-mail: :shark: :atom: :atom: :atom:

**Comment by zixuan75** [comment]
:-)

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.idea/
```
Comment (zixuan75) [inline .gitignore]: You cannot start Pycharm for connecting the `.idea` directory.

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1,3 @@
...
+.sh
```
Comment (zixuan75) [inline .gitignore]: Yes but it is a hidden file on linux :atom:

File: javascript/node-master.js (hunk 4 above)
Comment (zixuan75) [reply javascript/node-master.js]: The 'getMaster' attribute is only set for the outdated pull requests on the config.

File: javascript/node-master.js (hunk 3 above)
Comment (zixuan75) [reply javascript/node-master.js]: What did you mean? Is there a .getMaster file on your project?

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
This is the event comparison for #41.

File: .circleci/config.yml [hunk 9]
```diff
@@ -19,4 +19,18 @@ jobs:
     steps:
       - checkout
 
-      - echo "hello!"
+      - run: echo "hello world"
```
Comment (zixuan75) [inline .circleci/config.yml]: Echo Hello World

File: .circleci/config.yml [hunk 10]
```diff
@@ -19,4 +19,18 @@ jobs:
...
+  test:
+    docker: 
+      - image: circleci/node:7.10
+    working_directory: ~/repo
+    steps: 
+      - checkout
+      
+      - run: echo "hello this build step is to test this program using circleci"
```
Comment (zixuan75) [inline .circleci/config.yml]: Is this step to test?

**Comment by zixuan75** [comment]
# #42 and #41 is the issue called conflicting values. Also:
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
The #41 and #42 that the issue is [Change conflicting values (#42)](https://github.com/zixuan75/html_project/pulls).

**Comment by zixuan75** [comment]
@alax9 Your speed is @alax99 
@alax99 Your speed is @alax9 
# Speed who not commit to #41 
The #41 and #42 are intial issues that are solving problems.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
You need to touch your speed with #41.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
Preview **of the reply** is no solution to find #42.

**Comment by zixuan75** [comment]
Ok:
# Issue for event comparison (#41)
The [Change conflicting values (#42)](https://github.com/zixuan75/html_project/commit/89f5ae3affcd20b1b85100902543006feac3711d), [Change conflicting values (#41)](https://github.com/zixuan75/html_project/commit/ea9bc5e6aae73712b4c4379ff8383f852ef5ce93) is a issue that goes to a commit (It should be gone).

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
Preview **of the reply** now have solution to find #42.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
The #60 is closed as unfunny issue.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
#66 is the next simple project.

**Comment by zixuan75** [comment]
Ok:
# Issue for event comparison (#41)
The [Change conflicting values (#42)](https://github.com/zixuan75/html_project/commit/89f5ae3affcd20b1b85100902543006feac3711d), [Change conflicting values (#41)](https://github.com/zixuan75/html_project/commit/ea9bc5e6aae73712b4c4379ff8383f852ef5ce93) is a issue that goes to a commit (It should be gone).

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
Preview **of the reply** is going to leave these hour.

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
Preview **of the reply** is going to leave #1 .

**Comment by zixuan75** [comment]
Ok:
# Issue for event comparison (#41)
The [Change conflicting values (#42)](https://github.com/zixuan75/html_project/commit/89f5ae3affcd20b1b85100902543006feac3711d), [Change conflicting values (#41)](https://github.com/zixuan75/html_project/commit/ea9bc5e6aae73712b4c4379ff8383f852ef5ce93) is a issue that goes to a commit (It should be gone).

**Comment by zixuan75** [comment]
This is looking for :1st_place_medal:! Just make something to propose changes:

# Events comparison (#41)
I have #1 for #41 commit.

**Pull request merged**